sonar-batch/src/main/java/org/sonar/batch/bootstrap/GlobalContainer.java

```diff
@@ -53,8 +43,6 @@
 import org.sonar.core.util.DefaultHttpDownloader;
 import org.sonar.core.util.UuidFactoryImpl;
 
-import static java.lang.String.format;
-
 public class GlobalContainer extends ComponentContainer {
 
   private static final org.sonar.api.utils.log.Logger LOG = Loggers.get(GlobalContainer.class);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "LOG" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1068)